Podpora dockeru pro produkční nasazení (#9)
Misha12/Inkluzitron: README.md

## Produkční deployment

> TBD

Reviewer: Nerozjebe to teď leteronův PR #4? Se tam bude míchat čeština a angličtina.

Author: Ano. Rozbije. Aby se to vyřešilo, tak bude nutné prvně sloučit #4 a poté upravit rozdíly v tomto PR.

## Production deployment

> Docker is recommended for production deployment. The `src/` directory contains the necessary files to create docker image.

Reviewer 2: ```suggestion
Using Docker is recommended for production deployment. All files necessary can be found in the `src` directory.
```

### Local build

> 1) Switch to `src/` directory (`cd src/`).

Reviewer 2: ```suggestion
1) Enter the `src/` directory (`cd src/`).
```

> 2) Copy the `environment.template.env` file and name it `environment.env` and fill the required values.

Reviewer 2: ```suggestion
2) Create a copy of the `environment.template.env` file named `environment.env`, and fill the required values inside.
```

> 3) Run `docker-compose up`. The bot should automatically build and run.

Reviewer 2: ```suggestion
3) Run `docker-compose up`. The bot should be automatically built and run.
```

### DockerHub image

> Prepare environment file (first two steps in local build chapter) and run these commands:

Reviewer 2: ```suggestion
Prepare your environment file (first two steps in section Local build), and then run these commands:
```
